evidence/reactor.go

```diff
@@ -90,6 +90,9 @@ func (evR *Reactor) Receive(chID byte, src p2p.Peer, msgBytes []byte) {
 			case nil:
 				continue
 			default:
+				if err == ErrEvidenceAlreadyStored {
+					evR.Logger.Debug("Evidence already exists", "evidence", msg.Evidence)
```

Comment:
you forgot return OR continue

Reply:
or else

Reply:
more consistent if I make `ErrEvidenceAlreadyStored` as a type instead of a variable